<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Upplay</class>
 <widget class="QMainWindow" name="Upplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>561</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Upplay Player</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../.designer/backup/icons/play.png</normaloff>../../../../../.designer/backup/icons/play.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="PlayerHWidget" name="player_w" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="coverWidget" native="true"/>
        </item>
        <item>
         <widget class="QWidget" name="playlist_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>500</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="library_widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSelect_Media_Renderer"/>
    <addaction name="actionSelect_OH_Source"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Playlist"/>
    <addaction name="actionLoad_Playlist"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_viewLibrary"/>
    <addaction name="action_viewSearchPanel"/>
    <addaction name="action_smallPlaylistItems"/>
    <addaction name="separator"/>
    <addaction name="action_Dark"/>
    <addaction name="action_Fullscreen"/>
    <addaction name="separator"/>
    <addaction name="action_Preferences"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuFle"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="action_Close">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Dark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_viewLibrary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Library</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="action_Fullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="action_smallPlaylistItems">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Small Playlist Items</string>
   </property>
  </action>
  <action name="actionSelect_Media_Renderer">
   <property name="text">
    <string>Change Media Renderer</string>
   </property>
  </action>
  <action name="actionSelect_OH_Source">
   <property name="text">
    <string>Select Source</string>
   </property>
  </action>
  <action name="action_viewSearchPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search panel</string>
   </property>
  </action>
  <action name="actionSave_Playlist">
   <property name="text">
    <string>Save Playlist</string>
   </property>
  </action>
  <action name="actionLoad_Playlist">
   <property name="text">
    <string>Load Playlist</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlayerHWidget</class>
   <extends>QWidget</extends>
   <header>GUI/playerwidget/playerhwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
